site/content/docs/5.2/forms/layout.md

```diff
@@ -290,6 +318,13 @@ You can then remix that once again with size-specific column classes.
 
 Use the `.row-cols-*` classes to create responsive horizontal layouts. By adding [gutter modifier classes]({{< docsref "/layout/gutters" >}}), we'll have gutters in horizontal and vertical directions. On narrow mobile viewports, the `.col-12` helps stack the form controls and more. The `.align-items-center` aligns the form elements to the middle, making the `.form-check` align properly.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This form variant can not be used because it does not respect the Orange Design System specifications. The form text input label should always be placed above the input field.
```

Comment:
For what I see, there's only one visible label. I think the explanation must be rethought here.

Reply:
I don't understand "For what I see, there's only one visible label. I think the explanation must be rethought here"...

Reply:
New message:
This form variant should not be used because it does not respect the Orange Design System specifications. 

For the correct design, please refer to unitary [Forms](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459) components and [Pages](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53) form examples on the Orange Design System website.